x/airdrop/keeper/params_test.go

```diff
@@ -0,0 +1,54 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/airdrop/types"
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"testing"
+	"time"
+)
+
+var (
+	AllowedAddress  = sdk.AccAddress([]byte("allowed_____________"))
```

Comment:
when starting with a capital letter, those variables are both readable and writable from any other packages. In some cases, that can be required. But most of the time it must be avoided, as it induces potential side effects if some other code decides to modify its value. 

To be safe, let's make it private and local to this package.

```suggestion
	allowedAddress  = sdk.AccAddress([]byte("allowed_____________"))
```